test/creator.cpp

```diff
@@ -96,7 +96,7 @@ void test_redirect_dirent(
 TEST(ZimCreator, createEmptyZim)
 {
   unittests::TempFile temp("emptyzimfile");
-  auto tempPath = temp.path() + ".zim";
+  auto tempPath = temp.path();
   writer::Creator creator;
   creator.startZimCreation(tempPath);
```

Comment:
So, I had overlooked this fictitious usage of `TempFile` in #493, which resulted in the temporary ZIM file being left behind/not deleted. There is nothing to do about this comment, just documenting my own negligence in order to prevent some of it in the future.